### PR TITLE
LINK-1743 | Accessibility statement page

### PR DESCRIPTION
**Pull request by jorilindell**
## Description
A new page to display accessibility statement.
Accessibility statement -link to the footer

## Closes
[LINK-1743](https://helsinkisolutionoffice.atlassian.net/browse/LINK-1743)

## Screenshot
Screenshot 2023-11-28 at 16 03 18
Screenshot 2023-11-28 at 16 03 31

[LINK-1743]: https://helsinkisolutionoffice.atlassian.net/browse/LINK-1743?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/domain/accessibilityStatement/AccessibilityStatement.tsx [hunk 1]
```diff
@@ -0,0 +1,519 @@
...
+              niistä meille ja teemme parhaamme puutteiden korjaamiseksi.
+              Vastaamme 14 päivän kuluessa.
+            </p>
+            <p>
+              Voit ottaa yhteyttä{' '}
+              <Link to={`/${locale}${ROUTES.SUPPORT_CONTACT}`}>
+                verkkolomakkeella
+              </Link>{' '}
+              tai sähköpostitse osoitteella{' '}
+              <a href={`mailto:${SUPPORT_EMAIL}`}>${SUPPORT_EMAIL}</a>
```
Comment (mikkojamG) [inline src/domain/accessibilityStatement/AccessibilityStatement.tsx:229]: Extra $ sign:
Screenshot 2023-12-01 at 11 12 00

File: src/domain/accessibilityStatement/AccessibilityStatement.tsx [hunk 2]
```diff
@@ -0,0 +1,519 @@
...
+            <p>
+              We are constantly striving to improve the accessibility of the
+              online service. If you observe any issues that are not described
+              on this page, please let us know and we will do our best to fix
+              any shortcomings. We will respond to requests and comments as soon
+              as possible, but no later than within two weeks.
+            </p>
+            <p>
+              You can contact us by sending an email to{' '}
+              <a href={`mailto:${SUPPORT_EMAIL}`}>${SUPPORT_EMAIL}</a> / via an{' '}
```
Comment (mikkojamG) [inline src/domain/accessibilityStatement/AccessibilityStatement.tsx:65]: Extra $ sign:
Screenshot 2023-12-01 at 11 13 12

**Review by mikkojamG — CHANGES_REQUESTED** [review]

File: src/domain/accessibilityStatement/AccessibilityStatement.tsx (hunk 1 above)
Comment (jorilindell) [reply src/domain/accessibilityStatement/AccessibilityStatement.tsx]: Good catch. Removed the extra $.

File: src/domain/accessibilityStatement/AccessibilityStatement.tsx (hunk 2 above)
Comment (jorilindell) [reply src/domain/accessibilityStatement/AccessibilityStatement.tsx]: Good catch. Removed the extra $.

**Pull request merged**